### ISSUE TITLE
Can we have an environment variable to exclude grabbing of certain metrics?

### ISSUE THREAD

**Issue by cjwong1098** [qonto/prometheus-rds-exporter#15]
I am getting rds aws API throttled which is effecting gaps in metrics.
Whenever the app gets RDS API throttled, i don't think it returns metrics for RDS which results in metric gaps.

The exact issue I am facing is when the app is calling "DescribeDBLogFiles". I think this API only lets you target 1 instance at a time meaning that the more db instances one has, the more times you will have to call that api for each individual instance, which easily ends up getting API throttled by AWS.

I am guessing for many, there are a lot of metrics one may not need, for example in my case I do not need "rds_instance_log_files_size_bytes" (DescribeDBLogFiles). 

So it be nice if I can specify to exclude some metrics so that I don't I have to scrape it in general, but also I won't be throttled making those API calls.

Example error logs
```
{"time":"2023-10-16T11:43:37.941397651Z","level":"ERROR","msg":"can't scrape metrics: can't fetch RDS metrics: can't compute instance metrics for MYDBINSTANCE_REDACTED: can't get log files size for 824644574064: can't describe db logs files for MYDBINSTANCE_REDACTED: operation error RDS: DescribeDBLogFiles, exceeded maximum number of attempts, 3, https response error StatusCode: 400, RequestID: REDACTED, api error Throttling: Rate exceeded"}
```

**Comment by vmercierfr** [comment]
Hello,

It makes sense. We considered the possibility of choosing metrics to collect in the initial phase, but we didn't need to for our use case and postponed it.

We are using [Cobra](https://github.com/spf13/cobra), so we may implement it with environment variables and configuration files at the same time.

We may add a flag to enable/disable the following information:

- RDS Logs size (AWS RDS API)
- RDS quota (AWS quota API)
- RDS usage (AWS Cloudwatch API)
- RDS instance metrics (AWS Cloudwatch API)
- Instance type information (AWS EC2 API)

Do you need other parameters?

**Comment by cjwong1098** [comment]
Hi,
Thanks for getting back so soon. I think for now that looks like a good list.

My main concern is the RDS Logs size, so if I can disable the app from calling the DescribeDBLogFiles api that would be fantastic. That is specifically the api that is causing the throttle issue.  FYI i manually deleted some code to that uses the DescribeDBLogFiles api to see if the throttling stops and it looks like it did. 

As for the rest of the list, that looks pretty good as I may or may not want them. So if I don't need those metrics it be good to not have to make additional / unnecessary calls.

**Comment by vmercierfr** [comment]
Yes, definitely. It should be available by mid-next week while we design and ship it.

**Comment by vmercierfr** [comment]
I created the `control-collected-metrics` branch (https://github.com/qonto/prometheus-rds-exporter/pull/21) with parameters to control the different metrics to collect.

If you want to test it, you can use:

```
git clone git@github.com:qonto/prometheus-rds-exporter.git
cd prometheus-rds-exporter
git checkout control-collected-metrics
PROMETHEUS_RDS_EXPORTER_COLLECT_LOGS_SIZE=false go run . --debug
```

**Comment by cjwong1098** [comment]
I tried it out and it works!
this is perfect! thank you so much

**Comment by vmercierfr** [comment]
@cjwong1098 we just released the [0.2.8](https://github.com/qonto/prometheus-rds-exporter/releases/tag/0.2.8) that contains the new parameters.
